src/auth-service/utils/email.templates.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
 `;
   },
 
-  v2_emailVerification: (firstName, user_id, token) => {
+  v2_emailVerification: (email,firstName, user_id, token) => {
```

Comment:
Hi @BenjaminSsempala , under which use case is this being used?
Which endpoint?

Reply:
> Hi @BenjaminSsempala , any quick thoughts on this?
> Do we do any input validation for this email field from the user?

Hey @Baalmart , Yes the input validation is done in the form, which also has it's own script
Here's a link to the form  and the appscript
[Form](https://docs.google.com/forms/d/16AQre4iALbWD6MSd8cTZdHw5hAOt6oPf8zXmNkRoeV0/edit#responses)
[Appscript](https://script.google.com/u/2/home/projects/11W4BGQ_zIvOtBpUGGPz9ZAheBBTYvR5UWpHNrQIe-RCX6zofG9wRP8lD)

Reply:
thanks @BenjaminSsempala